lisa/sut_orchestrator/aws/platform_.py

```diff
@@ -125,6 +137,27 @@ class AwsPlatformSchema:
     wait_delete: bool = False
 
     def __post_init__(self, *args: Any, **kwargs: Any) -> None:
+        strip_strs(
+            self,
+            [
+                "aws_access_key_id",
+                "aws_secret_access_key",
+                "aws_session_token",
+                "aws_default_region",
+                "security_group_name",
+                "key_pair_name",
+                "locations",
+                "log_level",
+            ],
+        )
+
+        if self.aws_access_key_id:
+            add_secret(self.aws_access_key_id, mask=PATTERN_GUID)
```

Comment:
If it's not GUID format, it doesn't need to specify the mask as GUID. It can be left as default.